doc/source/whatsnew/v0.20.0.txt

```diff
@@ -539,6 +539,7 @@ Bug Fixes
 
 - Bug in ``resample``, where a non-string ```loffset`` argument would not be applied when resampling a timeseries (:issue:`13218`)
 
+- Bug in ``.groupby`` where ```.groupby(categorical, sort=False)`` would raise ``ValueError`` due to non-matching categories (:issue:`13179`)
```

Comment:
I meant obvious as in the combination of `sort=False` and a missing category in the data causing:

> ValueError: items in new_categories are not the same as in old categories

There is no other change in behavior except this now works as expected?